config/local.json5

```diff
@@ -0,0 +1,11 @@
+{
+  app: {
+    devMode: true,
+    checkInterval: 10,
+    logLevel: 5,
+    workdir: '/tmp/server'
```

Comment:
```suggestion
    workdir: './tmp/server'
```
Or are we planning to use the `tmp` system folder?

Reply:
yup, because that one is automatically deleted